src/scripts.js

```diff
@@ -43,78 +43,113 @@ let bookings;
 const loadPage = () => {
   getData()
   .then((data) => {
-    customer = new Customer(data[0].customers[0]);
     bookings = new Bookings(data[1].bookings)
   })
   hide([homeBtn])
+  todaysDateCalendar ();
 };
 
 const getData = () => {
   return Promise.all([fetchApiData('customers'), fetchApiData('bookings')])
 };
 
+const customerLogin = (event) => {
+  event.preventDefault();
+  disableLoginButton();
+  getData()
+  .then((data) => { 
+    let newCustomer = data[0].customers.find((elem) => (usernameText.value === `customer${elem.id}`) && (passwordText.value === "overlook2021"))
+    customer = new Customer(newCustomer)
+  })
+  .then(() => {
+    displayCustomerRoomDashboard()
+  }) 
+}
+
+const bookThisRoomPostApi = (roomNumber) => {
+  let newRoomNumber = parseInt(roomNumber)
+  fetch('http://localhost:3001/api/v1/bookings', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      "userID": customer.id,
+      "date": today,
+      "roomNumber": newRoomNumber,
+    })
+  })
+    .then(response => displaySuccessMessage(response))
+    .catch(err => {
+      displayServerErrorMessage(err)
+    });
+  displayRoomHasBeenBooked();
+}
+
+const displaySuccessMessage = () => {
```

Comment:
If you have time and you can see if you can remove some of your logic from domUpdates into scripts. If you have time.🙃